AllReadyApp/Web-App/AllReady/Areas/Admin/ViewModels/ManagerInvite/ManagerInviteDetailsViewModel.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
```

Comment:
out of curiosities sake, why did you use inheritance for these ViewModels? I can see the perceived benefit of "re-usability" but as I said earlier in some of my review comments, we want to minimize ViewModel sharing as much as possible b/c it leads to very tightly coupled code.

You can keep it for now, but I just wanted to know why. Thanks.